app/pulp/app/tasks/tags.py

```diff
@@ -1 +1,2 @@
 RESOURCE_REPOSITORY_TYPE = 'repository'
```

Comment:
I suggest to move `tags.py` out of the `pulp.app.tasks` because it may be expected that only tasks will be imported from `pulp.app.tasks`. Let's discuss where to put it.